okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -478,8 +494,7 @@ private void sanitizeAuthHeaders(MultiMap headers) {
    */
   private void relayToRequest(HttpClientResponse res, ProxyContext pc,
                               ModuleInstance mi) {
-    if (XOkapiHeaders.FILTER_AUTH.equals(mi.getRoutingEntry().getPhase())
-        && res.headers().contains(XOkapiHeaders.MODULE_TOKENS)) {
```

Comment:
This seems right. We shouldn't be controlling things based on the presence or absence of these headers.